data/src/main/java/co/nimblehq/compose/crypto/data/model/response/CoinItemResponse.kt

```diff
@@ -30,9 +35,41 @@ data class CoinItemResponse(
     val marketCapChange24h: BigDecimal?,
     @Json(name = "market_cap_change_percentage_24h")
     val marketCapChangePercentage24h: Double?,
+    @Json(name = "circulating_supply")
+    val circulatingSupply: BigDecimal?,
+    @Json(name = "total_supply")
+    val totalSupply: BigDecimal?,
+    @Json(name = "max_supply")
+    val maxSupply: BigDecimal?,
+    @Json(name = "ath")
+    val ath: BigDecimal?,
+    @Json(name = "ath_change_percentage")
+    val athChangePercentage: BigDecimal?,
```

Comment:
`percentage` should be in Double 🤔

Reply:
Agree 👍 We can use Double for `percentage`